src/chain-definition-custom/chain_def_testnet_poa_v0.1.0.json

```diff
@@ -2,7 +2,7 @@
   "name": "DataHighway Testnet",
```

Comment:
this file has been renamed to chain_def_testnet_v0.1.0.json
it's necessary to pull latest from master, then merge/rebase with the branch on your local machine and push changes to the branch of this PR
i.e.
git pull --rebase origin master
git checkout testnet/christian/DHX-145
git merge master
git push origin testnet/christian/DHX-145